backend/src/routes.ts

```diff
@@ -212,7 +213,7 @@ if (config.experimental.v2) {
   server.post('/api/v2/model/:modelId/files/upload/multipart/finish', ...postFinishMultipartUpload)
   server.delete('/api/v2/model/:modelId/files/:fileId', ...deleteFile)
 
-  // *server.get('/api/v2/model/:modelId/images', ...getImages)
+  server.get('/api/v2/model/:modelId/images', ...getImages)
```

Comment:
Is this obvious enough we are referring to Docker images?

Reply:
We could rename it `getOCIImages`, but that looks weird!